docs/MO_DG/prepare_model/Additional_Optimizations.md

```diff
@@ -61,19 +61,6 @@ the following commands to provide data in the ``NCHW`` layout:
    mo --input_model tf_nasnet_large.onnx --source_layout nhwc --target_layout nchw
    mo --input_model tf_nasnet_large.onnx --layout "nhwc->nchw"
 
-
-Again, if a model has more than one input or needs both input and output layouts 
-specified, you need to provide the name of each input or output to apply the layout.
-
-For example, to provide data in the ``NHWC`` layout for the `Yolo v3 Tiny` model 
-mentioned earlier, use the following commands:
-
-.. code-block:: sh
-
-   mo --input_model yolov3-tiny.onnx --source_layout "input_1(nchw),image_shape(n?)" --target_layout "input_1(nhwc)"
-   mo --input_model yolov3-tiny.onnx --layout "input_1(nchw->nhwc),image_shape(n?)"
-
-
```

Comment:
why is it removed?

Reply:
Please just add additional code block `python` to demonstrate also Python API. So we will have `sh` and `python`

Reply:
> why is it removed?

Again, please, refer to the ticket: 109837. See the  "Embedding Preprocessing Computation.pdf" file attached there, and requirements specified by the creator of this ticket.

Reply:
The content has been reinserted and `python` code block added with the https://github.com/openvinotoolkit/openvino/pull/17454/commits/d4c2cdfac6fd12a25f10c75c54129967af2465e1 commit